java-wasm-allocator/src/main/java/org/m4d/adp/WasmInterface.java

```diff
@@ -10,8 +12,12 @@
 
     public static final boolean LOADED_EMBEDDED_LIBRARY;
     static {
-        Path p = Paths.get("../wasm_interface/target/release/libwasm_interface.so");
-        System.load(p.toAbsolutePath().toString());
+        try {
+            System.out.println("nativeLoader");
+            NativeLoader.loadLibrary("wasm_interface");
+        } catch (IOException e) {
```

Comment:
Is it possible to make it throw instead of catching?

Reply:
As I saw [here](https://www.tutorialspoint.com/how-to-throw-an-exception-from-a-static-block-in-java), a `static` block cannot throw exceptions.